js/back.js

```diff
@@ -65,6 +65,38 @@
 
         return accumulator;
       }, {});
+      if (browsingDataObj.cookies) {
+        // delete cookies here + ignore whitelisted cookies
+        getStorage("cookieWhitelist", function(data) {
+          var domain_list = data.cookieWhitelist
+          getAllCookies({}, function(cookies)
+          {
+            var callbackCount = 0;
+            for (var i = 0; i < cookies.length; i++)
+            {
+              var cookie = cookies[i];
+              var url = getUrl(cookie.domain, cookie.path, cookie.secure);
+
+              var wl = false;
+              for (var x in domain_list) { 
+                var wl_domain = domain_list[x][0]
```

Comment:
Nit picking: I have doing nit picking, but I have to as I don't have linters setup yet(my bad ). May I ask you to use camelCase for variables, as it's done in other places as well.